### PR TITLE
Updated image and language to match new UI

### PR DESCRIPTION
**Pull request by nataliefiann**
## What are you changing in this pull request and why?

I've created this PR to update the images to match the current UI and also amended the language (as part of the global UI project) gear menu > account settings

## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).

---
🚀 Deployment available! Here are the direct links to the updated files:

- https://docs-getdbt-com-git-nfiann-faq-and-gear-dbt-labs.vercel.app/faqs/Environments/delete-environment-job
- https://docs-getdbt-com-git-nfiann-faq-and-gear-dbt-labs.vercel.app/faqs/Git/git-migration
- https://docs-getdbt-com-git-nfiann-faq-and-gear-dbt-labs.vercel.app/faqs/Git/github-permissions
- https://docs-getdbt-com-git-nfiann-faq-and-gear-dbt-labs.vercel.app/faqs/Git/gitignore
- https://docs-getdbt-com-git-nfiann-faq-and-gear-dbt-labs.vercel.app/faqs/Project/delete-a-project

### CODE REVIEW

File: website/docs/faqs/Environments/delete-environment-job.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ To delete a job or multiple jobs in dbt Cloud:
 4. Scroll to the bottom of the page and click **Delete** to delete the job. <br />
 
 <figure>
-<img src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/delete-job.jpg"/>
+<img src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/delete-job.png" width="100%"/>
```
Comment (mirnawong1) [inline website/docs/faqs/Environments/delete-environment-job.md]: non blocker but while we're here, maybe we can update to lightbox here instead? 
```suggestion
<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/delete-job.png" width="100%" title="Delete a job"/>
```

Comment (mirnawong1) [reply website/docs/faqs/Environments/delete-environment-job.md]: if you change, remember to move the figure tags (`<figure>`)

File: website/docs/faqs/Environments/delete-environment-job.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ Deleting an environment automatically deletes its associated job(s). If you want
 4. Scroll to the bottom of the page and click **Delete** to delete the environment. <br />
 
 <figure>
-<img src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/delete-environment.jpg"/>
+<img src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/delete-environment.png" width="100%"/>
```
Comment (mirnawong1) [inline website/docs/faqs/Environments/delete-environment-job.md]: same comment about lightbox above

File: website/docs/faqs/Git/gitignore.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ For more info on `gitignore` syntax, refer to the [Git docs](https://git-scm.com
 12. Once the branch has changed, click the **Pull from remote** button to pull in all the changes. 
 13. Verify the changes by making sure the files/folders in the `.gitignore `file are in italics.
```
Comment (mirnawong1) [inline website/docs/faqs/Git/gitignore.md]:
```suggestion
13. Verify the changes by making sure the files/folders in the `.gitignore` file are in italics. 
```

File: website/docs/faqs/Project/delete-a-project.md [hunk 4]
```diff
@@ -7,12 +7,12 @@ id: delete-a-project
 ---
 To delete a project in dbt Cloud, you must be the account owner or have admin privileges. 
 
-1. From dbt Cloud, click the gear icon at the top right corner and select **Account Settings**.
+1. From dbt Cloud, click on your account name in the left side menu and, select **Account settings**.
```
Comment (mirnawong1) [inline website/docs/faqs/Project/delete-a-project.md]:
```suggestion
1. From dbt Cloud, click on your account name in the left side menu and select **Account settings**.
```

**Review by mirnawong1 — APPROVED** [review]
thanks @nataliefiann !

File: website/docs/faqs/Environments/delete-environment-job.md (hunk 2 above)
Comment (nataliefiann) [inline website/docs/faqs/Environments/delete-environment-job.md]:
```suggestion
<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/delete-environment.png" width="100%"/> title="Delete an environment"/>
```

File: website/docs/faqs/Environments/delete-environment-job.md [hunk 5]
```diff
@@ -36,7 +36,7 @@ Deleting an environment automatically deletes its associated job(s). If you want
 4. Scroll to the bottom of the page and click **Delete** to delete the environment. <br />
 
 <figure>
```
Comment (nataliefiann) [inline website/docs/faqs/Environments/delete-environment-job.md]:
```suggestion

```

File: website/docs/faqs/Environments/delete-environment-job.md [hunk 6]
```diff
@@ -35,8 +35,7 @@ Deleting an environment automatically deletes its associated job(s). If you want
 3. Click **Settings** on the top right of the page and then click **Edit**.
 4. Scroll to the bottom of the page and click **Delete** to delete the environment. <br />
 
-<figure>
-<img src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/delete-environment.jpg"/>
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/delete-environment.png" width="100%"/> title="Delete an environment"/>
 <figcaption align = "center">Delete an environment</figcaption>
```
Comment (nataliefiann) [inline website/docs/faqs/Environments/delete-environment-job.md]:
```suggestion

```

File: website/docs/faqs/Environments/delete-environment-job.md [hunk 7]
```diff
@@ -35,9 +35,7 @@ Deleting an environment automatically deletes its associated job(s). If you want
 3. Click **Settings** on the top right of the page and then click **Edit**.
 4. Scroll to the bottom of the page and click **Delete** to delete the environment. <br />
 
-<figure>
-<img src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/delete-environment.jpg"/>
-<figcaption align = "center">Delete an environment</figcaption>
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/delete-environment.png" width="100%"/> title="Delete an environment"/>
 </figure>
```
Comment (nataliefiann) [inline website/docs/faqs/Environments/delete-environment-job.md]:
```suggestion

```

File: website/docs/faqs/Environments/delete-environment-job.md [hunk 8]
```diff
@@ -35,10 +35,7 @@ Deleting an environment automatically deletes its associated job(s). If you want
 3. Click **Settings** on the top right of the page and then click **Edit**.
 4. Scroll to the bottom of the page and click **Delete** to delete the environment. <br />
 
-<figure>
-<img src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/delete-environment.jpg"/>
-<figcaption align = "center">Delete an environment</figcaption>
-</figure>
+<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/delete-environment.png" width="100%"/> title="Delete an environment"/>
```
Comment (nataliefiann) [inline website/docs/faqs/Environments/delete-environment-job.md]:
```suggestion
<Lightbox src="/img/docs/dbt-cloud/cloud-configuring-dbt-cloud/delete-environment.png" width="100%" title="Delete an environment"/>
```

**Pull request merged**